### ISSUE TITLE
Support Paket for dotnet project

### ISSUE THREAD

**Issue by vbshift** [dependabot/feedback#363]
Do you plan to support Paket for dotnet project ?

Thank

**Comment by greysteil** [comment]
Yes! It's not on our immediate roadmap, but we definitely want to add support in future. I'm going to keep this open so others can watch it for updates, and add their voice to adding support.

**Comment by haf** [comment]
Since paket is a more mature way of handling dependencies for .net projects, having dependabot read `paket.lock` from the root would be really great!

**Comment by haf** [comment]
adfasdff
